packages/agent/src/dwn-manager.ts

```diff
@@ -532,4 +532,118 @@ export class DwnManager {
 
     return jws;
   }
-}
\ No newline at end of file
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
adding these newlines to create some distance between what we intend to keep around vs. the stuff we hope to refactor?

Reply:
Yes, that's the only reason they are there -- as a reminder to get rid of this patchwork of methods when we refactor.